README.md

```diff
@@ -7,6 +7,55 @@
 These are the officially supported images.</br>
 For the downstream / experimental images, check the [downstream](https://github.com/BredOS/images/tree/downstream) branch.</br>
 
+## Modifying the images to add new device
+We may want to add support for a new device or add support for a temperature or touch screen sensor on the i2c bus. This is different from the x86 world where the kernel is able to detect the hardware automatically with ACPI, etc. In the ARM world, the kernel needs to be told explicitly about the hardware. 
```

Comment:
There is no need to avoid explaining the device tree, we can use it to refer to all the changes as alterations of it.
x86 modifications also require device tree overrides, it'd be best not to use it as a comparison point.
```suggestion
To define hardware onto the kernel (like for example to add a temperature sensor) the kernel on ARM systems needs to be explicitly told about the hardware using the kernel device tree.
```